src/quotes/quotes.js

```diff
@@ -0,0 +1,148 @@
+const DEVOPS_LEAF = {
+  [PAST]: [
+    'completed fixing ingress path aliasing in periodically scheduled jobs.',
+    'launched a successful migration of skew-library for node annotations.',
+    'decoupled intermediate layers of web-proxify-echo container to lessen in-memory footprints.',
+    'applied global csrf rules as a secret exposure dependency.',
+    'debugged pods health probes resolution in prometheus namespace.',
+    'set missing routing multiplication for secondary VLAN gateway hops.'
+  ],
+  [FUTURE]: [
+    'reapply to reapply context recapture for discarded jobs.',
+    'reevaluate distributed label state synchronization for running pods.',
+    'verify if Jenkins pipeline flows operate respecting new image build scenarios.',
+    'verify Ansible output of the concatenation to diagnose failed ruling file.',
+    'resolve issues related to namespace targeting as a potential roadblock.',
+    'reestablish the routing table disparities to resolve hop debounces.'
+  ]
+};
+
+const BACKEND_LEAF = {
+  [PAST]: [
+    'finished throwing atomicized errors in nonpast conditions.',
+    'refactored module switching scheduler affinities for dereferenced items.',
+    'redfined Actor methods signatures for HTTP state tracking.',
+    'obtained race condition debugging scopes in inner maintainer loops.',
+    'collapsed curried parameters in Retry Monad Transformers.',
+    'reintegrated database schema validation dynamic libraries.'
+  ],
+  [FUTURE]: [
+    'reevaluate application of RESNET120 deep neural network for log data analysis.',
+    'restore conditionals for inferior module uploads ad-hoc.',
+    'derefence failed requests mid Redis session sustaining.',
+    'reread and reason on debugger flip states.',
+    'try mapping applicative switchers to recover from runtime errors.',
+    'configure the managment chaining of query structures.'
+  ]
+};
+
+const FRONTEND_LEAF = {
+  [PAST]: [
+    'repopulated bundling dependencies in repository structures. ',
+    'concretized treeshakeing entities in unportable packages.',
+    'refitted component properties into lifting abstractions.',
+    'unnested doctype declarations into iterative conditionals.',
+    'refactored dynamically changing view contexts in cross-browser compilations.',
+    'flattened stylesheet selectors for performant domain-scale compression.'
+  ],
+  [FUTURE]: [
+    'reinitiate state-view reducers depending on flow constraints.',
+    'fabricate animation effects as N-matrix transformations.',
+    'resupply parameters for inline form enforcements.',
+    'relocate implicit relative pathing to cross-domain CDN structures.',
+    'adapt in-browser interface zooming as native SPA operation.',
+    'phase out mutating data dependencies out of context-heavy external hooks.'
+  ]
+};
+
+export const ROLES = {
+  [DEV_OPS_ROLE]: DEVOPS_LEAF,
+  [BACK_END_ROLE]: BACKEND_LEAF,
+  [FRONT_END_ROLE]: FRONTEND_LEAF,
+  [PREFIX]: PREFIX_LEAF
+};
+
+export const ADVERBS = [
```

Comment:
I would put adverbs, adjectives and nouns in separate file too. Would be nice to explain their purpose too (that they are used to generate urls).